lib/routes/utilities/pushReplicationMetric.js

```diff
@@ -8,18 +6,15 @@ function shouldPushMetric(prevObjectMD, newObjectMD) {
     if (newObjectMD.getReplicationStatus() !== 'REPLICA') {
         return false;
     }
-    try {
-        // Replication of object tags and ACLs should not increment metrics.
-        assert.deepStrictEqual(prevObjectMD.getAcl(), newObjectMD.getAcl());
-        assert.deepStrictEqual(prevObjectMD.getTags(), newObjectMD.getTags());
-    } catch (e) {
-        return false;
-    }
-    return true;
+    // The rule for a REPLICA is simple: push object metrics if it is
+    // a new object version i.e. there is no existing object of the
+    // same version, which happens when updating ACLs and/or tags
+    // in-place, in which case we don't want to update object metrics.
```

Comment:
I read an article before I joined scality about writing docs and what you might want to not do. ( Will try and find it as well ). One of the things it said was not to use were phrases like "simple" or "its easy" as it makes those who do not understand it feel worse. Which could lead to lots of things, imposter syndrome is a common one. I know this is sort of inline code docs but I think it still applies.

Reply:
This might be the article I am referring to... https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher

Reply:
Ha, good one, I can change this easily :slightly_smiling_face:

Reply:
I removed "simple" and rephrased a bit the comment.